index.d.ts

```diff
@@ -0,0 +1,56 @@
+declare module testableUtils {
+    namespace log {
+        function fatal(message: any);
+        function info(message: any);
+        function error(message: any);
+        function debug(message: any);
+        function trace(message: any);
+    }
+
+    interface events {
+        addListener(event: string | symbol, listener: (...args: any[]) => void): this;
+        on(event: string | symbol, listener: (...args: any[]) => void): this;
+        once(event: string | symbol, listener: (...args: any[]) => void): this;
+        removeListener(event: string | symbol, listener: (...args: any[]) => void): this;
+        off(event: string | symbol, listener: (...args: any[]) => void): this;
+        removeAllListeners(event?: string | symbol): this;
+        setMaxListeners(n: number): this;
+        getMaxListeners(): number;
+        listeners(event: string | symbol): Function[];
+        rawListeners(event: string | symbol): Function[];
+        emit(event: string | symbol, ...args: any[]): boolean;
+        listenerCount(type: string | symbol): number;
+        // Added in Node 6...
+        prependListener(event: string | symbol, listener: (...args: any[]) => void): this;
+        prependOnceListener(event: string | symbol, listener: (...args: any[]) => void): this;
+        eventNames(): Array<string | symbol>;
+    }
+
+    function execute(func: Function)
+
+    namespace dataTable {
+        function open()
+        namespace open{
+            function get(i: number);
+            function random();
+            function next(options?: object);
+        }
```

Comment:
Is this declaring the response type of the `open()` function? Should it be a top level type here maybe?

I think we should declare the response types of the methods:

```
function get(i: number): Promise<DataRow>
function random(): Promise<DataRow>
function next(options?: object): Promise<Array<DataRow>>

//...

namespace DataRow {
  index: number;
  data: object;
  indexed: Array<any>;
}
```

Or something like that.